src/main/java/com/ait/lienzo/client/core/shape/Text.java

```diff
@@ -671,6 +822,41 @@ public Text setTextBaseLine(TextBaseLine baseline)
         return asAttributes(Attribute.TEXT, Attribute.FONT_SIZE, Attribute.FONT_STYLE, Attribute.FONT_FAMILY, Attribute.TEXT_UNIT, Attribute.TEXT_ALIGN, Attribute.TEXT_BASELINE);
     }
 
+    public BoundingBox getWrapBoundaries() {
+        return wrapBoundaries;
+    }
+
+    public Text setWrapBoundaries(BoundingBox boundaries, LayoutContainer.Layout layout) {
```

Comment:
Hey @Christopher-Chianelli , this part should be moved to the `wires` package. The `LayoutContainer` and the `Layout` are specific classes for using in wiires, so from my point of view, the core lienzo shapes should not depend on this.